### PR TITLE
Added v1.27 deprecations and removals blog 

### PR DESCRIPTION
**Pull request by harshitasao**
Added v1.27 deprecations and removals blog

### CODE REVIEW

**Comment by harshitasao** [comment]
cc @salaxander @mickeyboxell  @harshanarayana

File: content/en/blog/_posts/2023-03-16-kubernetes-1.27-deprecations-and-removals.md
Comment (sftim) [inline content/en/blog/_posts/2023-03-16-kubernetes-1.27-deprecations-and-removals.md:100]:
```suggestion
The v1.25 release of Kubernetes promoted `endPort` in NetworkPolicy to GA. NetworkPolicy providers that support the `endPort` field that can be used to specify a range of ports to apply a NetworkPolicy. Previously, each NetworkPolicy could only target a single port. So the feature gate `NetworkPolicyEndPort` will be removed in this release.
```

Please be aware that `endPort` field must be supported by the Network Policy provider. If your provider does not support endPort, and this field is specified in a Network Policy, the Network Policy will be created covering only the port field (single port)

File: content/en/blog/_posts/2023-03-16-kubernetes-1.27-deprecations-and-removals.md
Comment (sftim) [inline content/en/blog/_posts/2023-03-16-kubernetes-1.27-deprecations-and-removals.md:104]:
```suggestion
For a pod that is part of a StatefulSet, Kubernetes can mark the Pod ready only if Pod is available
(and passing checks)
for at least the period you specify in [`minReadySeconds`](/docs/concepts/workloads/controllers/statefulset/#minimum-ready-seconds). The feature became generally available in Kubernetes v1.25, and the
`StatefulSetMinReadySeconds` feature gate will be locked to true and removed in the v1.27 release.
```

File: content/en/blog/_posts/2023-03-16-kubernetes-1.27-deprecations-and-removals.md
Comment (sftim) [inline content/en/blog/_posts/2023-03-16-kubernetes-1.27-deprecations-and-removals.md:106]:
```suggestion
### Removal of `IdentifyPodOS` feature gate
```

File: content/en/blog/_posts/2023-03-16-kubernetes-1.27-deprecations-and-removals.md
Comment (sftim) [inline content/en/blog/_posts/2023-03-16-kubernetes-1.27-deprecations-and-removals.md:108]:
```suggestion
You can specify the operating system for a Pod, and the feature support for that is stable since
the v1.25 release.
The `IdentifyPodOS` feature gate will be removed for Kubernetes v1.27.
```

File: content/en/blog/_posts/2023-03-16-kubernetes-1.27-deprecations-and-removals.md
Comment (sftim) [inline content/en/blog/_posts/2023-03-16-kubernetes-1.27-deprecations-and-removals.md:110]:
```suggestion
### Removal of `DaemonSetUpdateSurge` feature gate
```

File: content/en/blog/_posts/2023-03-16-kubernetes-1.27-deprecations-and-removals.md
Comment (sftim) [inline content/en/blog/_posts/2023-03-16-kubernetes-1.27-deprecations-and-removals.md:112]:
```suggestion
The v1.25 release of Kubernetes also stabilised surge support for DaemonSet pods, implemented
in order to minimize DaemonSet downtime during rollouts. The `DaemonSetUpdateSurge` feature
gate will be removed in Kubernetes v1.27.
```

File: content/en/blog/_posts/2023-03-16-kubernetes-1.27-deprecations-and-removals.md
Comment (sftim) [inline content/en/blog/_posts/2023-03-16-kubernetes-1.27-deprecations-and-removals.md:126]: I'd drop these two as they are out of docs support.
```suggestion
```

File: content/en/blog/_posts/2023-03-16-kubernetes-1.27-deprecations-and-removals.md
Comment (sftim) [inline content/en/blog/_posts/2023-03-16-kubernetes-1.27-deprecations-and-removals.md:138]:
```suggestion
For information on the process of deprecation and removal, check out the official Kubernetes [deprecation policy](/docs/reference/using-api/deprecation-policy/#deprecating-parts-of-the-api) document.
```

File: content/en/blog/_posts/2023-03-16-kubernetes-1.27-deprecations-and-removals.md
Comment (sftim) [inline content/en/blog/_posts/2023-03-16-kubernetes-1.27-deprecations-and-removals.md:10]:
```suggestion
As Kubernetes develops and matures, features may be deprecated, removed, or replaced with better ones for the project's overall health. Based on the information available at this point in the v1.27 release process, which is still ongoing and can introduce additional changes, this article identifies and describes some of the planned changes for the Kubernetes v1.27 release.
```

(mid-cycle was actually late February)

File: content/en/blog/_posts/2023-03-16-kubernetes-1.27-deprecations-and-removals.md
Comment (sftim) [inline content/en/blog/_posts/2023-03-16-kubernetes-1.27-deprecations-and-removals.md:40]:
```suggestion
```
I'm afraid that that sentence doesn't make sense to me: what are these in addition to?

File: content/en/blog/_posts/2023-03-16-kubernetes-1.27-deprecations-and-removals.md
Comment (sftim) [inline content/en/blog/_posts/2023-03-16-kubernetes-1.27-deprecations-and-removals.md:49]:
```suggestion

Kubernetes v1.27 is not removing any other APIs; however several other aspects are going 
to be removed. Read on for details.

```

File: content/en/blog/_posts/2023-03-16-kubernetes-1.27-deprecations-and-removals.md
Comment (sftim) [inline content/en/blog/_posts/2023-03-16-kubernetes-1.27-deprecations-and-removals.md:50]:
```suggestion
### Support for deprecated seccomp annotations
```

File: content/en/blog/_posts/2023-03-16-kubernetes-1.27-deprecations-and-removals.md
Comment (sftim) [inline content/en/blog/_posts/2023-03-16-kubernetes-1.27-deprecations-and-removals.md:64]:
```suggestion
`ExpandCSIVolumes`
: Enable expanding CSI volumes.

`ExpandPersistentVolumes`
: Enable expanding PersistentVolumes.

`ExpandInUsePersistentVolumes`
: Enable expanding in-use PersistentVolumeClaims.
```

**Review by sftim — CHANGES_REQUESTED** [review]
Thanks.

This needs some changes before it is OK to publish. I hope this feedback helps you fix the things that aren't yet right.

**Comment by sftim** [comment]
/hold
Needs a future publication date.

File: content/en/blog/_posts/2023-03-17-kubernetes-1.27-deprecations-and-removals.md
Comment (sftim) [inline content/en/blog/_posts/2023-03-17-kubernetes-1.27-deprecations-and-removals.md:1]: (nit)
After this merges, we could move all the CLI argument removals to come before all the feature gate removals.

**Review by sftim — APPROVED** [review]
Thanks!

Let's publish it.
/lgtm
/approve

File: content/en/blog/_posts/2023-03-17-kubernetes-1.27-deprecations-and-removals.md
Comment (sftim) [inline content/en/blog/_posts/2023-03-17-kubernetes-1.27-deprecations-and-removals.md:4]:
```suggestion
date: 2023-03-17T14:00:00+0000
```

**Comment by sftim** [comment]
I forgot to unhold this earlier.

/lgtm
/approve
/hold cancel

**Pull request merged**
